independent-publisher-connectors/Proposal - ConnectWise (Independent Publisher)/apiDefinition.swagger.json

```diff
@@ -0,0 +1,393 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "ConnectWise",
+    "description": "ConnectWise API Connector - This connector is designed to aid in the Creation, Updating, and Selection of records from ConnectWise"
```

Comment:
Please remove the keyword "API" and "Connector" from the description.

Reply:
These requests have been done and I verified no errors in the swagger using Power Platform Custom Connector - View Swagger